widgetastic_manageiq/__init__.py

```diff
@@ -1356,11 +1356,13 @@ def find_row_on_pages(self, table, *args, **kwargs):
                                          .format(kwargs, table))
 
 
-def PaginationPane():  # noqa
-    return VersionPick({
-        Version.lowest(): NonJSPaginationPane(),
-        '5.9': JSPaginationPane(),
+def PaginationPane(*args, **kwargs):  # noqa
+    parent = kwargs.get("parent")
+    verpick_obj = VersionPick({
+        Version.lowest(): NonJSPaginationPane(*args, **kwargs),
+        '5.9': JSPaginationPane(*args, **kwargs),
     })
+    return verpick_obj.pick(parent.product_version) if parent else verpick_obj
```

Comment:
Of course, if you have a simple view that is not nested in any view then parent will always be a browser. But I would like it to be universal :)